Formula/amfora.rb

```diff
@@ -0,0 +1,38 @@
+class Amfora < Formula
+  desc "Fancy terminal browser for the Gemini protocol"
+  homepage "https://github.com/makeworld-the-better-one/amfora"
+  url "https://github.com/makeworld-the-better-one/amfora.git",
+      tag:      "v1.8.0",
+      revision: "71385e9f4e91ab982076856cac95320173ccea73"
+  license "GPL-3.0-or-later"
+  head "https://github.com/makeworld-the-better-one/amfora.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GO111MODULE"] = "on"
```

Comment:
```suggestion
```

With go1.16, go modules is turned on by default.

Reply:
Golang is obviously not my strongest skill :D